templates/quotes/blocks/quote.html

```diff
@@ -0,0 +1,9 @@
+{% load wagtailcore_tags %}
```

Comment:
I could be wrong but I think when outputting `StreamField` content  you only need to load `wagtailcore_tags` when iterating over the data

Reply:
It's needed for `include_block`